llvm/lib/Target/AMDGPU/SIRegisterInfo.cpp

```diff
@@ -1658,7 +1658,9 @@ void SIRegisterInfo::buildSpillLoadStore(
       MIB.addReg(SOffset, SOffsetRegState);
     }
 
-    int64_t CPol = AMDGPU::isGFX12Plus(ST) && LastUse ? AMDGPU::CPol::TH_LU : 0;
+    assert((!LastUse || AMDGPU::isGFX12Plus(ST)) &&
+           "last_use operand exists only on GFX12+");
```

Comment:
verifier should probably be checking this instead

Reply:
Removed / no longer using extra operand